test/integration/scheduler/scheduler_test.go

```diff
@@ -60,8 +60,8 @@ func TestSchedulerCreationFromConfigMap(t *testing.T) {
 
 	ns := framework.CreateTestingNamespace("configmap", s, t)
 	defer framework.DeleteTestingNamespace(ns, s, t)
-
-	clientSet := clientset.NewForConfigOrDie(&restclient.Config{Host: s.URL, ContentConfig: restclient.ContentConfig{GroupVersion: &schema.GroupVersion{Group: "", Version: "v1"}}})
+	kubeConfig := &restclient.Config{Host: s.URL, ContentConfig: restclient.ContentConfig{GroupVersion: &schema.GroupVersion{Group: "", Version: "v1"}}}
+	clientSet := clientset.NewForConfigOrDie(kubeConfig)
```

Comment:
Are these tests used by the scheduler plugins repo?
I prefer we leave this unchanged if we are not going to use the kubeconfig.

Reply:
Reverted as these tests don't use the kube config.